src/solver/hydro/management/HydroInputsChecker.cpp

```diff
@@ -50,27 +46,24 @@ void HydroInputsChecker::Execute(uint year)
 {
     prepareInflows_.Run(year);
     minGenerationScaling_.Run(year);
-
     if (!checksOnGenerationPowerBounds(year))
     {
-        throw FatalError("hydro inputs checks: invalid minimum generation");
+        std::ostringstream msg;
+        msg << "hydro inputs checks: invalid minimum generation in year " << year;
+        errorCollector_.Collect(msg.str());
```

Comment:
```suggestion
        errorCollector_ << "hydro inputs checks: invalid minimum generation in year " << year;
```
Who doesn't like a bit of syntactic sugar ?